docs/multicluster/user-guide.md

```diff
@@ -559,6 +560,42 @@ In future releases, some additional tooling may become available to automate the
 creation of ResourceExports for ACNPs, and provide a user-friendly way to define
 Multi-cluster NetworkPolicies to be enforced in the ClusterSet.
 
+## Multicluster Pod to Pod Connectivity
+Since Antrea v1.9.0, Multi-cluster supports routing Pod traffic across clusters through the Multi-cluster Gateways.
+Pod IPs can be reached in all member clusters within a ClusterSet. If you want to enable this feature,
+the cluster's Pod CIDRs must be set in ConfigMap `antrea-mc-controller-config-**` of each member cluster like below example.
```

Comment:
We should change to active voice to be consistent with the former sentence: "you should set the cluster's Pod CIDRs in ConfigMap ..."

Or change the first sentence to "To enable this feature"

Reply:
"like the example below"